lib/core-integration/src/Test/Integration/Scenario/API/Wallets.hs

```diff
@@ -177,6 +177,37 @@ spec = do
             , expectFieldNotEqual passphraseLastUpdate Nothing
             ]
 
+    it "CREATE_WALLET_01 - SQL injection attempt when creating a wallet" $ \ctx -> do
```

Comment:
Maybe, for better recognition, we could tag those tests following the OWASP naming (or ordering) of particular risks (from https://owasp.org/www-project-api-security/). So for the injection it could be:
 - `OWASP_INJECTION_CREATE_WALLET` or
 - `OWASP_API8_CREATE_WALLET`

Reply:
Also, maybe it makes sense to add similar injection tests for some other requests that have string type payload parameters?
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/putWallet
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/putWalletPassphrase
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postByronRandomWallet (and others, i.e. icarus, trezor, ledger)
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransactionFee
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/migrateByronWallet
 - https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/joinStakePool

Reply:
will switch to `OWASP_INJECTION_CREATE_WALLET`

Reply:
@piotr-iohk when it comes to other injection tests...I played with a number of endpoints last week when researching - was not able to have any malfunction. mainly because we use `persistent`'s `insert_`, `repsert`,... all of them use proper escaping... which is very good. we use in one case, in very controlled, safe way, `rawExecute`